pydantic/types.py

```diff
@@ -28,39 +29,42 @@
     email_validator = None
 
 __all__ = [
-    'NoneStr',
-    'NoneBytes',
-    'StrBytes',
-    'NoneStrBytes',
-    'StrictStr',
-    'ConstrainedBytes',
-    'conbytes',
-    'ConstrainedStr',
-    'constr',
-    'EmailStr',
-    'UrlStr',
-    'urlstr',
-    'NameEmail',
-    'PyObject',
-    'DSN',
-    'ConstrainedInt',
-    'conint',
-    'PositiveInt',
-    'NegativeInt',
-    'ConstrainedFloat',
-    'confloat',
-    'PositiveFloat',
-    'NegativeFloat',
-    'ConstrainedDecimal',
-    'condecimal',
-    'UUID1',
-    'UUID3',
-    'UUID4',
-    'UUID5',
-    'FilePath',
-    'DirectoryPath',
-    'Json',
-    'JsonWrapper',
+    "NoneStr",
```

Comment:
please revert all these quotes changes and use single quotes in all code.

Reply:
Sorry I had used black formatter the wrong way. 
I've just reverted these changes